android/build.gradle

```diff
@@ -75,10 +77,10 @@ uploadArchives {
     repositories.mavenDeployer {
         beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-        repository(url: sonatypeRepo) {
-            authentication(userName: whisperSonatypeUsername, password: whisperSonatypePassword)
-        }
-
+//        repository(url: sonatypeRepo) {
```

Comment:
This is for publishing to Signal's maven repo, which we won't be doing.

Reply:
Why not just delete the line?

Reply:
Do you mind leaving a comment to that effect then?

Reply:
(Same thing on the other similar block in java/build.gradle)